example/server/integrations/sep31.py

```diff
@@ -76,10 +76,10 @@ def process_post_request(
 
         transaction.amount_fee = round(
             transaction.asset.send_fee_fixed
+            or 0
```

Comment:
I don't think this does what you want. In Python `+` has higher precedence than `or`, so the `0 + ...` would occur together, but that doesn't matter, since the `or` early exits and only `.send_fee_fixed` is returned if it is set.

I think you want:
```python
(transaction.asset.send_fee_fixed or 0) + ...
```

Ref:https://docs.python.org/3/reference/expressions.html#operator-precedence